command.go

```diff
@@ -286,8 +285,7 @@ func (c *Command) HelpTemplate() string {
 		return c.parent.HelpTemplate()
 	} else {
 		return `{{with or .Long .Short }}{{. | trim}}{{end}}
-{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}
-`
```

Comment:
My output looks like this:

```
$ ./cockroach help

Usage:
  cockroach [command]

Available Commands:
  init        init new Cockroach cluster
  start       start a node by joining the gossip network
  cert        create ca, node, and client certs
  exterminate destroy all data held by the node
  quit        drain and shutdown node

  log         make log files human-readable

  sql         open a sql shell
  kv          get, put, conditional put, increment, delete, scan, and reverse scan key/value pairs
  user        get, set, list and remove users
  range       list, split and merge ranges
  zone        get, set, list and remove zones

  version     output version information
  help        Help about any command

Flags:
      --alsologtostderr=false: log to standard error as well as files
      --color="auto": colorize standard error output according to severity
      --log-backtrace-at=:0: when logging hits line file:N, emit a stack trace
      --log-dir="": if non-empty, write log files in this directory
      --logtostderr=true: log to standard error instead of files
      --verbosity=0: log level for V logs
      --vmodule=: comma-separated list of file=N settings for file-filtered logging


Use "cockroach [command] --help" for more information about a command.
```
Notice that there's also a leading newline, which I would also like to get rid of, but not sure how.